Amazon.IonObjectMapper.Test/CustomIonSerializerTest.cs

```diff
@@ -13,28 +13,29 @@ public void SerializeObjectsWithCustomContextFactoryAttribute()
         {
             var customContext = new Dictionary<string, object>()
             { 
-                { "customCourseSerializer", new UpdateCourseSections()}
+                { "customCourseSerializer", new UpdateCourseSections() }
             };
             Check(TestObjects.bob, new IonSerializationOptions { CustomContext = customContext });
-            var customSerializer = new IonSerializer(new IonSerializationOptions { CustomContext = customContext});
+            var customSerializer = new IonSerializer(new IonSerializationOptions { CustomContext = customContext });
             var stream = customSerializer.Serialize(TestObjects.bob);
             var serialized = StreamToIonValue(stream);
             Assert.IsTrue(serialized.ContainsField("name"));
             Assert.IsTrue(serialized.ContainsField("course"));
             Assert.IsTrue(serialized.ContainsField("id"));
             var course = serialized.GetField("course");
             Assert.AreEqual(9, course.GetField("Sections").IntValue);
-            Assert.AreEqual(DateTime.Now.ToString(), course.GetField("MeetingTime").StringValue);
+            Assert.AreEqual(DateTime.Parse("2021-10-10T13:15:21Z").ToString(), course.GetField("MeetingTime").StringValue);
```

Comment:
Why not just have the string instead of parsing to datetime then back to string?

Reply:
Changed to use string directly.